src/main/java/homework2/user/Main.java

```diff
@@ -0,0 +1,15 @@
+package homework2.user;
+
+public class Main {
+    public static void main(String[] args) {
+
+        Client client  = new Client();
+        Admin admin = new Admin();
```

Comment:
the idea here is to use abstraction rather than implementation

Reply:
Yes first i did it. But in this case i cannot use methods getTicket & checkTicket. but if add theese methods for User it will brake SOLID principles

Reply:
????????????

Reply:
how you add it?

Reply:
try to use casting)